modules/ROOT/pages/vpn-high-availability.adoc

```diff
@@ -0,0 +1,42 @@
+= Anypoint VPN High Availability
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-disabled.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-disabled.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+To ensure your applications and related operations are tolerant to Anypoint VPN updates, issues, or individual Customer Gateway failures, implement High Availability VPN connections.
+
+Each Anypoint VPN connection consists of two endpoints, providing High Availability (HA) on the MuleSoft side by default.
+
+== Prerequisites
+
+* Sufficient entitlement for each VPN connection.
+* Two VPN endpoints using different public IP addresses.
+* Two VPN connections in Runtime Manager.
+
+== Configure High Availability with Anypoint VPN
+
+The example below shows a High Availability VPN topology using a single MuleSoft VPC and two VPN connections.
```

Comment:
The example below > The following example

MuleSoft VPC > Anypoint Virtual Private Cloud (Anypoint VPC) (use complete name first time in the narrative, then short)

According to the term presentation guide, this is called Anypoint VPC, not MuleSoft VPC (unless they are different things). Please confirm with devs.